doc/system_requirements.md

```diff
@@ -110,6 +110,20 @@ Covers:
 
 Needs: dsn
 
+#### Verify LICENSE File
+
+`req~verify-license-file~1`
+
+PK verifies that the project has a `LICENSE` file.
+
+If not, PK can create the file with a MIT license and copyright statement for the current year.
```

Comment:
Yes and no. I think you can split it up quite far, but since it's a simple feature I prefer to have it in one requirement and not write multiple requirements + design blocks for this feature.